app/cdap/components/ParsingDrawer/__test__/ParsingDrawer.test.tsx

```diff
@@ -15,22 +15,17 @@
  */
 
 import { fireEvent, render } from '@testing-library/react';
+import history from 'app/cdap/services/history';
 import React from 'react';
-import { createBrowserHistory as createHistory } from 'history';
 import { Route, Router, Switch } from 'react-router';
 import ParsingDrawer from '..';
-
-const history = createHistory({
-  basename: '/',
-});
-
 describe('It Should Test the Parsing Drawer Component', () => {
   it('Should render the Parsing Drawer Parent Component', () => {
     const container = render(
       <Router history={history}>
         <Switch>
           <Route>
-            <ParsingDrawer />
+            <ParsingDrawer setLoading={() => jest.fn()} updateDataTranformation={() => jest.fn()} />
```

Comment:
you should define a function setLoading and use here
so you van asset if the function is being called or not
plz see if this makes sense and talk to me

same for updateDataTranformation

Reply:
aasign it to respective unit test reviewer